src/components/MessagesList/MessagesList.vue

```diff
@@ -243,6 +251,13 @@ export default {
 		chatIdentifier() {
 			return this.token + ':' + this.isParticipant + ':' + this.viewId
 		},
+
+		dateSeparatorsOrdered() {
+			const orderedList = [...this.dateSeparatorsList].sort((a, b) => {
+				return moment(a.date, 'YYYY-DDD').diff(moment(b.date, 'YYYY-DDD'))
+			})
+			return orderedList.map(item => item.dateSeparator)
```

Comment:
I made this array to not keep any risk of being unordered ( because soft update can change the order, if it is an old group it will be added to the end)